src/action/mod.rs

```diff
@@ -192,35 +207,34 @@ impl Action {
     }
 
     /// Purpose was to check, check complete, so print the results.
-    fn check(&self, suggestions_per_path: SuggestionSet, _config: &Config) -> Result<()> {
+    fn check(&self, suggestions_per_path: SuggestionSet, _config: &Config) -> Result<Finish> {
         let mut count = 0usize;
         for (_path, suggestions) in suggestions_per_path {
             count += suggestions.len();
             for suggestion in suggestions {
                 println!("{}", suggestion);
             }
         }
-        if count > 0 {
-            Err(anyhow::anyhow!(
-                "Found {} potential spelling mistakes",
-                count
-            ))
-        } else {
-            Ok(())
-        }
+        Ok(Finish::MistakeCount(count))
```

Comment:
So there is no message about how many mistakes were found right? Intentional?

Reply:
Yes, we could, but it's already handled, in `Action::check` and in `Action::interactive` it's kind of pointless (or at least not implemented).

Reply:
I just noticed it, I also don't think it's necessary :)